src/cryptsetup/cryptsetup-generator.c

```diff
@@ -99,11 +99,13 @@ static int split_keyspec(const char *keyspec, char **ret_keyfile, char **ret_key
         return 0;
 }
 
-static int generate_keydev_mount(
+static int generate_dev_mount(
                 const char *name,
-                const char *keydev,
-                const char *keydev_timeout,
+                const char *dev,
+                const char *type_prefix,
```

Comment:
maybe add a brief comment here to the right side what this is, i.e. `/* "keydev-" or "header-" */` to make this easier to grok

Reply:
Ok, fixed